server.js

```diff
@@ -15,12 +15,97 @@ const app = express();
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
+const listEndpoints = require('express-list-endpoints')
+
+const { Schema } = mongoose;
+const HappyThoughtSchema = new Schema({
+  message: {
+    // type is the most important one
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+    // removes unnecessary whitespaces from string
+    trim: true
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date, 
+    default: Date.now
+  }
+})
+
+const HappyThought = mongoose.model("HappyThought", HappyThoughtSchema);
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(listEndpoints(app));
```

Comment:
Nice job on adding the listEndpoints to the first page of the API! It looks great.